src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2856,6 +2876,15 @@ static bool createEntity
     return false;
   }
 
+  /* Search for a potential date expiration attribute */
+  mongo::Date_t dateExpiration;
+
+  if (!processDateExpirationAtEntityCreation(attrsV, &dateExpiration, errDetail, oe))
```

Comment:
If `oe` can be changed by `processDateExpirationAtEntityCreation`, it should be sent as a "C" pointer. That's the convention we use in the broker.
Read only objects are sent as C++ references (`const XXX& obj`) and objects that are to be modified are sent as C pointers.
This way, you see in the call that the called function may alter the object sent to it.

Read all about it in the style guide (contribution guide):
https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/contribution_guidelines.md

Reply:
`oe` is a pointer to `OrionError` (line 2848) so I  think is ok.

Reply:
Didn't see that.  So, `oe` should be renamed to `oeP` and it will be clear.
Imagine we have something about this in the style guide

Reply:
Ok, I noticed that we have the same situation in removeEntity, updateContextAttributeItem, appendContextAttributeItem, deleteContextAttributeItem, processContextAttributeVector, processContextAttributeVector. So I guess they should be fixed (maybe in a dedicated issue?)

Reply:
I think that by the moment use the rigth variable name in the part touched by this PR is enough. However, if you want to create an issue about the missing ones for the future, please go ahead :)

Reply:
Fixed in 0ee2585